packages/canary-web-components/src/utils/types.ts

```diff
@@ -301,3 +301,5 @@ export enum IcWeekDays {
   Friday = 5,
   Saturday = 6,
 }
+
+export type IcPosition = "top" | "right";
```

Comment:
Can we make this more specific, eg IcPositionTopOrRight or even IcTableOfContentsPosition? IcPosition sounds like a name that could be useful more generally across other components in future.  This way it would not be a breaking change if we want to have an IcPosition further down the line which also supports "bottom" or "left"